src/entities/stabilityTracker/ui/StabilityTracker.gyro.backup.js

```diff
@@ -0,0 +1,412 @@
+  const {
+    onChange,
+    config: initialConfig,
+    maxLambda, // comes from redux in legacy
+    // appletId,
+    onComplete,
+  } = props;
+
+  const config = {
+    lambdaSlope: initialConfig?.lambdaSlope || 190.0,
+    durationInMinutes: initialConfig?.durationInMinutes || 1, //5,
+    numberOfTrials: initialConfig?.numberOfTrials || 2,
+    phase: initialConfig?.phase ?? 'trial',
+  };
+
+  const targetPoints = generateTargetTrajectory(
+    config.durationInMinutes,
+    TASK_LOOP_RATE,
+    1,
+    center,
+  );
+
+  const [isMoving, setMoving] = useState(false);
+  // const [tickNumber, setTickNumber] = useState(0);
+  const [userInputType, setUserInputType] = useState(
+    initialConfig?.userInputType ?? 'touch',
+  );
+  const [score, setScore] = useState < number > 0;
+  const numberOfTrials = useRef(0);
+  const userPosition = useRef < Coordinate > CENTER_COORDINATES;
+  const circlePosition = useRef < Coordinate > CENTER_COORDINATES;
+  const [tick, setTickNumber] = useState < number > 0;
+  const boundWasHit = useRef(false);
+  const lambdaSlope = useRef(config.lambdaSlope);
+  const boundHitAnimationDuration = useRef(0);
+  const responses = useRef([]);
+  const showControlBar = useRef(true);
+  const gyroListener = useRef();
+  const baseOri = useRef();
+  const lambdaValue = useRef(INITIAL_LAMBDA);
+  const startPosition = useRef(0);
+  const IS_TOUCH = useMemo(() => userInputType === 'touch', [userInputType]);
+  const IS_TRIAL = config.phase === 'trial';
+
+  const lambdaLimit = IS_TRIAL ? 0 : maxLambda * 0.3;
+
+  const updateUserPosition = (x: number, y: number) => {
+    userPosition.current = [x, y];
+
+    userPosition.current[1] = Math.max(0, userPosition.current[1]);
+    userPosition.current[1] = Math.min(
+      userPosition.current[1],
+      PLAYGROUND_WIDTH - BLOCK_HEIGHT * 2 + OUTER_CIRCLE_RADIUS * 2,
+    );
+    userPosition.current[1] += BLOCK_HEIGHT - OUTER_CIRCLE_RADIUS;
+  };
+
+  const onUserStartedMoving = (event: GestureResponderEvent) => {
+    if (IS_TOUCH) {
+      startPosition.current = event.nativeEvent.locationY;
+    }
+  };
+
+  const onUserStoppedMoving = (event: GestureResponderEvent) => {
+    if (IS_TOUCH) {
+      if (isMoving) {
+        updateUserPosition(
+          event.nativeEvent.locationX,
+          center + event.nativeEvent.locationY - startPosition.current,
+        );
+      } else {
+        updateUserPosition(
+          event.nativeEvent.locationX,
+          event.nativeEvent.locationY,
+        );
+      }
+      startPosition.current = 0;
+    }
+    setMoving(true);
+
+    showControlBar.current = false;
+  };
+
+  const onUserMove = (event: GestureResponderEvent) => {
+    if (IS_TOUCH && !showControlBar.current) {
+      updateUserPosition(
+        event.nativeEvent.locationX,
+        center + event.nativeEvent.locationY - startPosition.current,
+      );
+    }
+  };
+
+  const finishResponse = () => {
+    onComplete(['atata']);
```

Comment:
The below - looks like temp code.
Can we remove it? Or change somehow to get it prepared for merge to dev